lib/howitzer/web/iframe_dsl.rb

```diff
@@ -100,6 +100,12 @@ def define_has_no_iframe(name, args)
             capybara_context.has_no_selector?(*iframe_element_selector(args, params))
           end
         end
+
+        def transform_name(name)
+          "#{name}_page".classify.constantize
+        rescue NameError
+          "#{name.to_s.split('_').collect(&:capitalize).join('::')}Page".constantize
```

Comment:
It does not cover all cases. What about this one:

```ruby
module Foo
  class MySuperPage < Howitzer::Web::Page
  end
  
 # in your case it will be
  # iframe :foo__my_super -> Foo::My::SuperPage
end
```
The easiest way to make all possible combinations and use define? instead of handling errors

Reply:
Also, do not forget to test module nested in module as well

Reply:
Please, consider to introduce second parameter as a class object to define class explicitly.

```
def iframe(name, *args)
  if args.first.is_a?(Class)
end  
```